homeassistant/components/sensor/broadlink.py

```diff
@@ -34,16 +36,26 @@
     'noise': ['Noise', ' '],
 }
 
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Optional(CONF_NAME, default=DEVICE_DEFAULT_NAME): vol.Coerce(str),
-    vol.Optional(CONF_MONITORED_CONDITIONS, default=[]):
-        vol.All(cv.ensure_list, [vol.In(SENSOR_TYPES)]),
-    vol.Optional(CONF_UPDATE_INTERVAL, default=timedelta(seconds=300)): (
-        vol.All(cv.time_period, cv.positive_timedelta)),
-    vol.Required(CONF_HOST): cv.string,
-    vol.Required(CONF_MAC): cv.string,
-    vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.positive_int
-})
+PLATFORM_SCHEMA = vol.All(
+    PLATFORM_SCHEMA.extend({
+        vol.Optional(CONF_NAME, default=DEVICE_DEFAULT_NAME): vol.Coerce(str),
+        vol.Optional(CONF_MONITORED_CONDITIONS, default=[]):
+            vol.All(cv.ensure_list, [vol.In(SENSOR_TYPES)]),
+        vol.Optional(CONF_UPDATE_INTERVAL):
+            vol.All(cv.time_period, cv.positive_timedelta),
+        vol.Optional(CONF_SCAN_INTERVAL, default=DEFAULT_INTERVAL):
```

Comment:
This is already part of the base schema. Instead we should define `SCAN_INTERVAL` constant which will set the default update interval.

Reply:
Makes sense.